File: autofin/src/main/java/model/basic_details/BasicDetails.java
package model.basic_details;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

import model.add_lead_details.Document;

public class BasicDetails {
    @SerializedName("documents")
    @Expose
    private List<Document> documents = null;
    @SerializedName("fullName")
    @Expose
    private String fullName;
    @SerializedName("email")
    @Expose
    private String email;
    @SerializedName("customerMobile")
    @Expose
    private String customerMobile;
    @SerializedName("otp")
    @Expose
    private String otp;
    @SerializedName("salutation")
    @Expose
    private String salutation;

    public List<Document> getDocuments() {
        return documents;
    }

    public void setDocuments(List<Document> documents) {
        this.documents = documents;
    }

    public String getFullName() {
        return fullName;
    }

    public void setFullName(String fullName) {
        this.fullName = fullName;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getCustomerMobile() {
        return customerMobile;
    }

    public void setCustomerMobile(String customerMobile) {
        this.customerMobile = customerMobile;
    }

    public String getOtp() {
        return otp;
    }

    public void setOtp(String otp) {
        this.otp = otp;
    }

    public String getSalutation() {
        return salutation;
    }

    public void setSalutation(String salutation) {
        this.salutation = salutation;
    }

}

File: autofin/src/main/java/v2/model/request/add_lead/BasicDetails.kt
package v2.model.request.add_lead

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
public class BasicDetails(

        @SerializedName("FirstName") public var FirstName: String? = null,
        @SerializedName("LastName") public var LastName: String? = null,
        @SerializedName("Email") public var Email: String? = null,
        @SerializedName("CustomerMobile") public var CustomerMobile: String? = null,
        @SerializedName("Salutation") public var Salutation: String? = null

) : Parcelable